explorer/mempool.go

```diff
@@ -147,11 +172,31 @@ func (exp *explorerUI) storeMempoolInfo() {
 	var totalOut float64
 	var totalSize int32
 
+	exp.NewBlockDataMtx.Lock()
+	lastBlock := exp.NewBlockData.Height
+	lastBlockHash, err := exp.blockData.GetBlockHash(lastBlock)
+	if err != nil {
+		log.Warnf("Could not get block hash for last block, mempool vote filter disabled")
+	}
+	lastBlockTime := exp.NewBlockData.BlockTime
+	exp.NewBlockDataMtx.Unlock()
```

Comment:
Since the above block of new code is identical to the code in `mempoolMonitor` (except for the message), please make a new `(exp *explorerUI)` method to get the lastBlock* stuff.